### PR TITLE
add responsive in rootlayout

### PR DESCRIPTION
**Pull request by jessieyu777**

### CODE REVIEW

**Review by jessieyu777 — COMMENTED** [review]
sidebar will hide under 768px and content will responsive in the page

File: packages/playground/src/layouts/RootLayout.tsx [hunk 1]
```diff
@@ -85,9 +85,9 @@ const RootLayout: React.FC = () => {
   ]
 
   return (
-    <div className="bg-background flex">
+    <div className="bg-background grid md:grid-cols-[220px_minmax(900px,_1fr)]">
```
Comment (abhinavrastogi-harness) [inline packages/playground/src/layouts/RootLayout.tsx:88]: maybe we should also add the class `min-w-screen` to ensure a min-width of 100vh, incase the content doesn't have enough width

File: packages/playground/src/layouts/PipelineLayout.tsx [hunk 2]
```diff
@@ -7,8 +7,8 @@ const PipelineLayout: React.FC = () => {
   const { pipelineId } = useParams<{ pipelineId: string }>()
 
   return (
-    <div>
-      <div style={{ display: 'flex', gap: '10px' }}>
+    <div className="h-[calc(100vh-100px)]">
+      <div className="flex p-2 gap-2.5 h-[44px] items-center">
```
Comment (abhinavrastogi-harness) [inline packages/playground/src/layouts/PipelineLayout.tsx:11]: why do we need a fixed height and `items-center` here? adding the correct vertical padding will fix both height and alignment.

File: packages/playground/src/pages/pipeline-edit-page.tsx [hunk 3]
```diff
@@ -275,7 +275,7 @@ export default function PipelineEditPage() {
   )
 
   return (
-    <div style={{ height: '650px' }}>
+    <div className="h-[calc(100vh-144px)]">
```
Comment (abhinavrastogi-harness) [inline packages/playground/src/pages/pipeline-edit-page.tsx:278]: can we remove this wrapper div and add the correct height on `Container.Root` directly by passing `className` to it?

**Comment by joetaylor-harness** [comment]
I'm running this locally and see the yaml/visual editor layout is improved, nice work. I don't see the navbar hide responsively.

File: packages/playground/src/layouts/PipelineLayout.tsx (hunk 2 above)
Comment (jessieyu777) [reply packages/playground/src/layouts/PipelineLayout.tsx]: fix all of them to work properly height with padding is 44px so I removed the 44px

File: packages/playground/src/pages/pipeline-edit-page.tsx (hunk 3 above)
Comment (jessieyu777) [reply packages/playground/src/pages/pipeline-edit-page.tsx]: Solved in the new commit

**Comment by jessieyu777** [comment]
> I'm running this locally and see the yaml/visual editor layout is improved, nice work. I don't see the navbar hide responsively.

Solve in the current commit, please check it out

**Review by jessieyu777 — COMMENTED** [review]
fix the layout and discuss them after meeting today

**Review by abhinavrastogi-harness — APPROVED** [review]

**Pull request merged**